pkg/apis/apps/v1beta1/zz_generated.conversion.go

```diff
@@ -291,7 +290,8 @@ func autoConvert_v1beta1_DeploymentSpec_To_extensions_DeploymentSpec(in *v1beta1
 		return err
 	}
 	out.Selector = (*meta_v1.LabelSelector)(unsafe.Pointer(in.Selector))
-	if err := api_v1.Convert_v1_PodTemplateSpec_To_api_PodTemplateSpec(&in.Template, &out.Template, s); err != nil {
+	// TODO: Inefficient conversion - can we improve it?
```

Comment:
this is unexpected... do we know why this is happening?

Reply:
it appears that the autogenerated conversions are getting created in a weird order, and since other packages use this type, they are failing to find the autogenerated exported one

Reply:
@liggitt thanks for digging into it. Are there any solutions for this?

Reply:
spent a while trying to fix the generator and was not successful. moving `Convert_v1_PodTemplateSpec_To_api_PodTemplateSpec` and `Convert_api_PodTemplateSpec_To_v1_PodTemplateSpec` from `zz_generated.conversion.go` to `conversion.go` and registering them as manually registered conversions (even though they would just delegate to the auto-generated conversion) lets other packages pick up the functions again.

```go
func Convert_v1_PodTemplateSpec_To_api_PodTemplateSpec(in *v1.PodTemplateSpec, out *api.PodTemplateSpec, s conversion.Scope) error {
	return autoConvert_v1_PodTemplateSpec_To_api_PodTemplateSpec(in, out, s)
}

func Convert_api_PodTemplateSpec_To_v1_PodTemplateSpec(in *api.PodTemplateSpec, out *v1.PodTemplateSpec, s conversion.Scope) error {
	return autoConvert_api_PodTemplateSpec_To_v1_PodTemplateSpec(in, out, s)
}
```

Reply:
@liggitt thank you so much for digging into this. I move them back as you suggested.